### ISSUE TITLE
Could we change the browser uri from item id to item handle?

### ISSUE THREAD

**Issue by CrisGuzmanS** [DSpace/DSpace#8838]
Hello @tdonohue and everyone 👋

**Describe the solution you'd like**
In DSpace 6 the item uri is  displayed according to the handle:
![image](https://github.com/DSpace/DSpace/assets/51240376/3a55859e-7c43-474c-9727-dfd908db0599)

but in Dspace 7 is displayed according to the item id:
![image](https://github.com/DSpace/DSpace/assets/51240376/70e5dc90-37f4-49d4-98ae-5157b6514e89)

i am able to replace **/items/id** to **/handle/handle** and it works, but i would like to instead of displaying the id the browser should displays the handle

**Comment by tdonohue** [comment]
@CrisGuzmanS : Currently, this is not possible in DSpace 7 & there are no plans to support this.

As you've noticed though, all older (v1-v6) DSpace URLs will automatically redirect to the proper location in DSpace 7.  So, if you visit a URL that has `/handle/[prefix]/[suffix]` in DSpace 7, it will redirect you to the new URL patter (e.g. it will redirect to `/items/[uuid]` if that handle referenced an Item).  So, Handles work well in DSpace 7, but they are no longer included in the URL.  (But the Handle URL will still appear on the Item page as shown in your DSpace 7 screenshot above)

We specifically changed the URL structure in DSpace 7 because we are (slowly) removing the requirement to use Handles. The old `/handle/[prefix]/[suffix]` URLs only made sense for institutions that chose to purchase & use Handles (from https://www.handle.net/).   We've now started to move to a model where Handles may still be used but they are _no longer required_.  Therefore, sites can choose to use them, or they could instead choose to use DOIs or similar.

We also discussed this older `/handle/[prefix]/[suffix]` URL pattern with the Google Scholar team a few years back.  At that time, they also recommended moving to a URL path structure which *describes* the type of content, as that is beneficial for indexing by Google & Google Scholar.  This is part of the reason why our URL patterns in DSpace 7 look like this:
* `/items/[uuid]`  (For a regular Item)
* `/entities/person/[uuid]` (For a Person Entity)
* `/collections/[uuid]` (for a Collection)
* `/communities/[uuid]` (for a Community)

In summary: at this time, there are no plans to support `/handle/[prefix]/[suffix]` URLs in DSpace 7 or later.  We've already setup automatic redirect though to ensure that old links will redirect to the new location after sites upgrade from DSpace v1-v6 to DSpace v7.

**Comment by CrisGuzmanS** [comment]
Thank you @tdonohue that is clear for me. i will close this issue
